scripts/e2e-tests.sh

```diff
@@ -82,33 +82,33 @@ function go_test_e2e() {
 # Download the k8s binaries required by kubetest.
 # Parameters: $1 - GCP project that will host the test cluster.
 function download_k8s() {
-  local version=${E2E_CLUSTER_VERSION}
   # Fetch valid versions
   local versions="$(gcloud container get-server-config \
       --project=$1 \
       --format='value(validMasterVersions)' \
       --region=${E2E_CLUSTER_REGION})"
   local gke_versions=(`echo -n ${versions//;/ /}`)
   echo "Valid GKE versions are [${versions//;/, }]"
-  if [[ "${version}" == "latest" ]]; then
+  if [[ "${E2E_CLUSTER_VERSION}" == "latest" ]]; then
     # Get first (latest) version, excluding the "-gke.#" suffix
-    version="${gke_versions[0]%-*}"
-    echo "Using latest version, ${version}"
-  elif [[ "${version}" == "default" ]]; then
+    E2E_CLUSTER_VERSION="${gke_versions[0]%-*}"
+    echo "Using latest version, ${E2E_CLUSTER_VERSION}"
+  elif [[ "${E2E_CLUSTER_VERSION}" == "default" ]]; then
     echo "ERROR: `default` GKE version is not supported yet"
     return 1
   else
-    echo "Using command-line supplied version ${version}"
+    echo "Using command-line supplied version ${E2E_CLUSTER_VERSION}"
   fi
   # Download k8s to staging dir
-  version=v${version}
+  version=v${E2E_CLUSTER_VERSION}
```

Comment:
Unused, remove.

Reply:
Actually, this line must be:

```suggestion
  E2E_CLUSTER_VERSION=v${E2E_CLUSTER_VERSION}
```

That's why the integration tests failed.